test/cell_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def setup
+    skip
+    @cell = Cell.new("B4")
+  end
+
+  def test_it_exists
+    skip
+    assert_instance_of Cell, @cell
+  end
+
+  def test_it_has_coordinates
+    skip
+    assert_equal "B4", @cell.coordinate
+  end
+
+  def test_it_does_not_have_a_ship
+    skip
+    assert_equal nil, @cell.ship
+  end
+
+  def test_it_is_empty
+    skip
+    assert @cell.empty?
+  end
+
+  def test_it_has_a_new_ship
+    skip
+    assert_equal @cruiser, @cell.place_ship
+  end
+
+  def test_it_is_not_empty
+    skip
+    refute @cell.empty?
+  end
+
+  def test_it_has_not_been_fired_upon
+    skip
+    refute @cell.fired_upon?
+  end
+
+  def test_when_hit_loses_1_health
+    @cruiser.hit
+    assert_equal 2, @cell.ship.health
+  end
+
+  def test_it_has_been_fired_upon
+    skip
+    assert @cell.fired_upon?
+  end
+
+  def test_cell_has_not_been_fired_upon
+    skip
+    assert_equal ".", cell_1.render
+  end
+
+  def test_fired_upon_and_shot_missed
+    skip
+    assert_equal "M", cell_1.render
+  end
+
+  def test_it_has_a_new_cell
+    skip
+    assert_equal cell_2, Cell.new("C3")
+  end
+
+  def test_cell_has_a_ship
+    skip
+    assert_equal @cruiser, cell_2.place_ship
```

Comment:
@amymoller 
Your test name is `test_it_has_a_ship`, but in this test, while we do want to place a ship, we are actually testing how the cell renders (aka: does it show a ". , S, H, or X"). 

So first we will want to place our ship with `cell_2.place_ship(cruiser)`, remembering to define `cell_2` and `cruiser` either in this test or in our setup method. 

Then we can call `cell_2.render` which should return `"."`, unless we use the optional argument `true` like so: `cell_2.render(true)` which means we will reveal that there is a ship there with an "S". 

Then we can test what happens after we fire on the cell (it should render an "H" because the ship was hit), and we can also check whether the ship has been `.sunk?` depending on how many times it was hit. 

I would encourage you to take another look at the last few lines of the interaction pattern and see how you can re-write the following tests to include the necessary method calls before each assert_equals line!